components/raftstore/src/store/worker/read.rs

```diff
@@ -437,10 +439,10 @@ impl ReadDelegate {
         false
     }
 
-    pub fn check_stale_read_safe<S: Snapshot>(
+    pub fn check_stale_read_safe<EK: KvEngine>(
```

Comment:
Wny need a generic?

Reply:
It may be used before, but now it is not needed.